### ISSUE TITLE
pyenv for (Django) deployment?

### ISSUE THREAD

**Issue by nicktimko** [yyuu/pyenv#315]
I've been trying to figure out if it's worth it to try to deploy a Django app with pyenv, and there seems to be conflicting information/suggestions, e.g. [this yyuu comment](https://github.com/yyuu/pyenv/issues/65#issuecomment-26155323) versus [this wiki page](https://github.com/yyuu/pyenv/wiki/Deploying-with-pyenv) that comes across as encouraging that "[s]etting up pyenv on a production server is exactly the same as in development"(!), then provides not that much direction for how to set it up for a daemon user, permissions, etc, etc.

Basically my question is, is it a passable idea to use pyenv for (Django) deployments?  The wiki page should probably be edited either way to be a little more clear on the issue.

**Comment by yyuu** [comment]
In [my previous comment](https://github.com/yyuu/pyenv/issues/65#issuecomment-26155323), I meant that (basically) I would not recommend to use pyenv-built python to dynamically link to external application servers such like Apache's `mod_wsgi`. It's technically possible and you can try such configuration. I think it's too much complicated, though.

I'd recommend you to use pyenv-built python for production deployment of python applications as it described in [the wiki page](https://github.com/yyuu/pyenv/wiki/Deploying-with-pyenv). As the answer for your question, for the deployment of Django applications, I can recommend you to use pyenv.
